### PR TITLE
Translate sibebar items

### PR DESCRIPTION
**Pull request by XiShanYongYe-Chang**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

Translate sibebar items.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by samzong** [comment]
提醒下，我们需要在 `Next` 版本上才可以看到效果 @windsonsea @XiShanYongYe-Chang 

<https://deploy-preview-521--karmada.netlify.app/zh/docs/next/userguide/clustermanager/cluster-registration/>

File: i18n/zh/docusaurus-plugin-content-docs/current.json [hunk 1]
```diff
@@ -3,21 +3,93 @@
...
+  "sidebar.docs.category.Multi-cluster Security Compliance Governance": {
+    "message": "多集群安全合规治理",
+    "description": "The label for category Multi-cluster Security Compliance Governance in sidebar docs"
+  },
+  "sidebar.docs.category.Best Production Practices": {
+    "message": "最佳生产实践",
+    "description": "The label for category Best Production Practices in sidebar docs"
+  },
+  "sidebar.docs.category.Autoscaling across clusters": {
+    "message": "跨集群自动缩放",
```
[the remark this answers is not in the document]
Comment (XiShanYongYe-Chang) [reply i18n/zh/docusaurus-plugin-content-docs/current.json]: 多谢检视，已修改。

**Review by RainbowMango — COMMENTED** [review]
/assign @samzong

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**